_rules/6cbcbe.md

```diff
@@ -0,0 +1,206 @@
+<a href="/test-assets/6cbcbe/index-copy.html">Contact us</a>
+```
+
+#### Passed example 4
+
+Same link text used for links going to pages where the content section is the same, but where the navigation options (bread crumbs and local sub menus) differ due to different placement in navigation hierarchy:
+
+```html
+<a href="/test-assets/6cbcbe/about/contact.html">Contact us</a>
+<a href="/test-assets/6cbcbe/careers/contact.html">Contact us</a>
+```
+
+#### Passed example 5
+
+URLs differ due to trailing slashes, but resolves to the same resource after redirects caused by user agent:
+
+```html
+<a href="/test-assets/6cbcbe/">Contact us</a> 
+<a href="/test-assets/6cbcbe">Contact us</a>
+```
+
+#### Passed example 6
+
+Pages contain different amounts of information and/or differently worded information, but fulfils same purpose in relation to the link:
+
+```html
+<a href="/test-assets/6cbcbe/page1.html">Call us</a>
+<a href="/test-assets/6cbcbe/page2.html">Call us</a>
+```
+
+#### Passed example 7
+
+Pages have the same advertised key content but use different layouts:
+
+```html
+<a href="/test-assets/6cbcbe/page1.html">Contact us</a>
+<a href="/test-assets/6cbcbe/page3.html">Contact us</a>
+```
+
+#### Passed example 8
+
+Links created via scripting with explicit role of link, but leads to the same resource: 
+
+```html
+<span role="link"
+   onclick="location='/test-assets/6cbcbe/index.html'">
+Link text
+</span>
+
+<span role="link"
+   onclick="location='/test-assets/6cbcbe/index.html'">
+Link text
+</span>
+```
+
+### Failed
+
+#### Failed example 1
+
+Same link text used for links going to different resources:
+
+```html
+<a href="http://facebook.com">Follow us</a> 
+<a href="http://twitter.com">Follow us</a>
+```
+
+#### Failed example 2
+
+Same link text used for links going to web pages that are similar, but has different information in the content:
+
+```html
+<a href="/test-assets/6cbcbe/about/contact.html">Contact us</a> 
+<a href="/test-assets/6cbcbe/admissions/contact.html">Contact us</a>
+```
+
+#### Failed example 3
+
+Links created via scripting with explicit role of link, but leads to different resources that offer different content: 
+
+```html
+<span role="link"
+   onclick="location='/test-assets/6cbcbe/about/contact.html'">
+Link text
+</span>
+
+<span role="link"
+   onclick="location='/test-assets/6cbcbe/admissions/contact.html'">
+Link text
+</span>
+```
+
+### Inapplicable 
+
+#### Inapplicable example 1
+
+´a´ and ´area´ elements without ´href´ attribute:
+
+```html
+<a>Link text</a>
+<area aria-label="Link text">
+```
+
+#### Inapplicable example 2
+
+No identical link texts:
+
+```html
+<a href="/test-assets/6cbcbe/about/contact.html">Contact main office</a>
+<a href="/test-assets/6cbcbe/admissions/contact.html">Contact admissions office</a>
+```
+
+#### Inapplicable example 3
+
+Link is not included in the accesssibility tree:
+
+```html
+<a href="/test-assets/6cbcbe/page1.html" aria-hidden="true">Contact Us</a>
```

Comment:
I'd put `tabindex="-1"` on this link, make sure the example is accessible.

Reply:
Tabindex"-1" added - even though test cases don't have to be accessible...